libsoftwarecontainer/src/container.cpp

```diff
@@ -190,9 +190,9 @@ bool Container::create()
     m_rootFSPath = buildPath(s_LXCRoot, containerID, "rootfs");
 
     if (m_enableWriteBuffer) {
-        const std::string rootFSPathLower = m_containerRoot + m_id + "/rootfs-lower";
-        const std::string rootFSPathUpper = m_containerRoot + m_id + "/rootfs-upper";
-        const std::string rootFSPathWork  = m_containerRoot + m_id + "/rootfs-work";
+        const std::string rootFSPathLower = m_containerRoot + "/rootfs-lower";
+        const std::string rootFSPathUpper = m_containerRoot + "/rootfs-upper";
+        const std::string rootFSPathWork  = m_containerRoot + "/rootfs-work";
```

Comment:
This seems to be unrelated with the checkContainerRoot stuff. Please split it into its own commit.